dp/cloud/docker/postgresql/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:13.3
```

Comment:
missing license header

Reply:
Do Dockerfile also have to include this?

Reply:
Yes, every file where you can add a header, should have an header

Reply:
Added